src/openrct2-ui/audio/AudioChannel.cpp

```diff
@@ -15,7 +15,6 @@
 #include <openrct2/audio/AudioChannel.h>
 #include <openrct2/audio/AudioSource.h>
 #include <openrct2/common.h>
-#include <openrct2/core/Math.hpp>
```

Comment:
Many of the files don't include <algorithm> directly now, which is where `std::clamp` is defined.

Reply:
Oh 🤦‍♂️, on it!